src/readDwarf.h

```diff
@@ -17,10 +17,12 @@ enum DebugLevel : unsigned {
 	DbgPdbTypes = 0x2,
 	DbgPdbSyms = 0x4,
 	DbgPdbLines = 0x8,
-	DbgDwarfTagRead = 0x10,
-	DbgDwarfAttrRead = 0x20,
-	DbgDwarfLocLists = 0x40,
-	DbgDwarfLines = 0x80
+	DbgPdbContrib = 0x10,
+	DbgDwarfTagRead = 0x100,
+	DbgDwarfAttrRead = 0x200,
+	DbgDwarfLocLists = 0x400,
+	DbgDwarfRangeLists = 0x800,
+	DbgDwarfLines = 0x1000

```

Comment:
It would probably be a better idea to keep the original values and add the new ones to the end.

Reply:
I want to group the flags by theme, so all of the dwarf flags are next to each other, and all the pdb flags are next to each other too.  In general, the higher numbered flags are much more verbose than the lower-numbeded flags.

The other thing to note about the DWARF debug printing is that it's meant to be used with the output of llvm-dwarfdump or objdump -W to see where we went wrong.